Decred Wallet/Custom Views/ContouredLabel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ContouredLabel.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+class ContouredLabel: UILabel {
```

Comment:
What is a `Contoured` Label? Perhaps, let's just use `Label` and leave this open to more customisations as we go.